skimage/io/tests/test_sift.py

```diff
@@ -29,9 +29,12 @@ def test_load_sift():
 13 12 75 119 35 0 0 13 28 14 37 121 12 0 0 21 46 5 11 93
 29 0 0 3 14 4 11 99''')
     f.close()
-    f = open(fname, 'rb')
-    features = load_sift(f)
-    f.close()
+
+    # Check whether loading by filename works
+    load_sift(fname)
+
+    with open(fname, 'rb') as f:
```

Comment:
I'm curious why do we test here in binary mode, while function uses text mode internally.